extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -303,8 +319,11 @@ void ignoreWarnings(BuildProducer<ReflectiveHierarchyIgnoreWarningBuildItem> ign
     @BuildStep
     void registerClasses(
             BuildProducer<NativeImageSystemPropertyBuildItem> nativeImageProps,
-            BuildProducer<ServiceProviderBuildItem> providerItem, final BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
-            final BuildProducer<NativeImageResourceBundleBuildItem> resourceBundle) {
+            BuildProducer<ServiceProviderBuildItem> providerItem,
+            BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
+            BuildProducer<NativeImageResourceBundleBuildItem> resourceBundle) {
+        reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, "com.sun.xml.bind.v2.ContextFactory"));
+        reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, "com.sun.xml.internal.bind.v2.ContextFactory"));
```

Comment:
Why are these now needed?

Reply:
This is due to a wrong merge on my side. Removed. Thanks for spotting!